pages/index.js

```diff
@@ -34,23 +41,62 @@ export default function Home() {
       </Head>
 
       <Main>
-        <ul>
-          {cardData.map((card) => {
-            return (
-              <Card
-                key={card.id}
-                name={card.name}
-                onDelete={() => deleteCard(card.id)}
-              />
-            );
-          })}
-        </ul>
+        <div>
+          <Headline>Today</Headline>
+          <HabitForTodayList>
+            {cardData
+              .filter((item) => !item.checked)
+              .map((card) => {
+                return (
+                  <Card
+                    key={card.id}
+                    name={card.name}
+                    onDelete={() => deleteCard(card.id)}
+                    onToggle={() => toggleCard(card.id)}
+                  />
+                );
+              })}
+          </HabitForTodayList>
+          <br />
```

Comment:
Why do you need a break element here? If it is for styling reasons, better solve it with CSS grid or flexbox.

Reply:
I have removed the <br />